frontend/src/metabase/query_builder/components/expressions/ExpressionMode.js

```diff
@@ -8,6 +8,15 @@ class ExpressionHighlight extends window.ace.acequire(
 
     this.$rules = {
       start: [
+        {
+          token: "aggregation",
+          regex:
+            "Average|CountIf|CumulativeCount|CumulativeSum|Median|Min|Max|Percentile|Share|StandardDeviation|SumIf|Variance",
+        },
+        {
+          token: "boolean",
+          regex: "between|contains|endsWith|interval|isempty|isnull|startsWith",
+        },
```

Comment:
This duplicates the information from `MBQL_CLAUSES` in `frontend/src/metabase/lib/expressions/config.js`.
If we can't keep it DRY, then let's not worry at all about coloring the functions based on its type. I don't think that's the end of the world, CMIIW @flamber.

Reply:
Thanks @ariya. 

@flamber What do you think?